internal/shim-sgx/src/enclave.rs

```diff
@@ -39,6 +39,7 @@ mod _internal {
     ///  If rax == 0, we are doing normal execution.
     ///  Otherwise, we are handling an exception.
     #[no_mangle]
+    #[allow(named_asm_labels)]
```

Comment:
We probably want to `warn` here instead of `allow`. This is one case where warning makes sense because we want to know when the problem is resolved.